samples/drivers/led_lp5562/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _lp5562:
+
+LP5562: 4-Channel RGB
+######################
+
+Overview
+********
+
+This sample controls 4 LEDs connected to a TI LP5562 driver, using the
+following pattern:
+
+ 1. turn on LEDs to be red
+ 2. turn on LEDs to be green
+ 3. turn on LEDs to be blue
+ 4. turn on LEDs to be white
+ 5. turn on LEDs to be yellow
+ 6. turn on LEDs to be purple
+ 7. turn on LEDs to be cyan
+ 8. turn on LEDs to be orange
+ 9. turn off LEDs
+ 10. blink the LEDs in white
+ 11. turn off LEDs
+ 12. blink the LEDs in purple
+ 13. turn off LEDs
+
+The sample assumes that RGB LEDs are connected in the intended colors to the
```

Comment:
Replace these two sentences with:

    Refer to the `LP5562 Manual`_ for the RGB LED connections and color
    channel mappings used by this sample.